projects/plugins/jetpack/extensions/blocks/videopress/edit.js

```diff
@@ -402,28 +402,31 @@ const VideoPressEdit = CoreVideoEdit =>
 				</Fragment>
 			);
 
-			if ( isUploading ) {
-				return (
-					<Fragment>
-						{ blockSettings }
-						<Loading text={ __( 'Uploading…', 'jetpack' ) } />
-					</Fragment>
-				);
-			}
+			const isFetching = ! isUploading && ( isFetchingMedia || isFetchingPreview );
+			const useCoreVideo = ! isUploading && ! isFetching && ( fallback || ! preview );
+			const useCoreVideoOrLoading = isUploading || isFetching || useCoreVideo;
 
-			if ( isFetchingMedia || isFetchingPreview ) {
+			// In order for the media placeholder to keep its state for error messages, we need to keep the CoreVideoEdit component in the tree
```

Comment:
```suggestion
			// In order for the media placeholder to keep its state for error messages, we need to keep the CoreVideoEdit component in the tree during file uploads
```

Just a comment suggestion to add a bit more clarity

Reply:
✅